### PR TITLE
Prevent endless loops and long running feed processing

### PR DESCRIPTION
**Pull request by annando**
I had a look at my workers today. There had been some really long running processes that blocked the whole system. It was caused by an endless loop while fetching AP items and some feeds with very much items.

### CODE REVIEW

File: static/defaults.config.php [hunk 1]
```diff
@@ -273,6 +273,10 @@
 		// Maximum number of queue items for a single contact before subsequent messages are discarded.
 		'max_contact_queue' => 500,
 
+		// max_feed_items (Integer)
+		// Maximum number of feed items that are fetched and processed. For unlimited items set to 0.
+		'max_feed_items' => 10,
```
Comment (MrPetovan) [inline static/defaults.config.php]: Will this pick up items where it left off the next time the feed is polled of there are more than 10 items?

Comment (annando) [reply static/defaults.config.php]: It is always fetching the newest 10 items.

Comment (MrPetovan) [reply static/defaults.config.php]: What is the number of items of the problematic feed? Can we raise this to 15 or 20 and still be within a reasonable performance threshold?

Comment (annando) [reply static/defaults.config.php]: Yeah, no problem with that. I had feeds with more than 200 items, *that* was intense.

**Pull request merged**